src/components/FieldSelect/index.tsx

```diff
@@ -1,16 +1,18 @@
-import React, { useEffect, useState } from 'react';
+import React, { ReactNode, useEffect, useState } from 'react';
 
 import { FieldProps, useField } from '@formiz/core';
-import { GroupBase } from 'react-select';
+import { GroupBase, SingleValue } from 'react-select';
 
 import { FormGroup, FormGroupProps } from '@/components/FormGroup';
 import { Select, SelectProps } from '@/components/Select';
 
+type MinimumOption = { value: unknown; label: ReactNode };
```

Comment:
Maybe we can extract this type declaration in a single place (it is also declared in the `<FieldMultiSelect />`)

Reply:
I don't know, I don't want to create dependencies between two components. I want an additional opinion on this